types/validation.go

```diff
@@ -40,6 +48,17 @@ func VerifyCommit(chainID string, vals *ValidatorSet, blockID BlockID,
 	// ignore all absent signatures
 	ignore := func(c CommitSig) bool { return c.BlockIDFlag == BlockIDFlagAbsent }
 
+	// attempt to verify aggregated commit
```

Comment:
After discussion, let's use `v.test` here to only accept aggregated commits in production